tests/views_integration/Tribe/Events/Views/V2/Rest_EndpointTest.php

```diff
@@ -0,0 +1,38 @@
+<?php
+namespace Tribe\Events\Views\V2;
+
+class Rest_EndpointTest extends \Codeception\TestCase\WPTestCase {
```

Comment:
Add a test to check for sanitization.
Add a test for the discrimination between WordPress REST API and TEC REST API (a test matrix).
Add some tests for the AJAX handling method to make sure it returns the same stuff the REST-based approach returns.

Reply:
What do you mean a test between TEC and WP rest API?